ibis/backends/duckdb/__init__.py

```diff
@@ -1534,3 +1534,16 @@ def insert(
                 table_name,
                 obj if isinstance(obj, pd.DataFrame) else pd.DataFrame(obj),
             )
+
+    def _get_temp_view_definition(self, name: str, definition: str) -> str:
```

Comment:
Does duckdb use these for DDL? At least you removed these methods for the other backends.

Reply:
Yes DuckDB uses these for all the read_foo methods.